### DISCUSSION TITLE
Github pages not working for a private repository for me!

### DISCUSSION THREAD

**Discussion by shwetasng** [community/community#59461] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I tried publishing my website made using HTML, CSS through github pages but since it is a private repository i am getting a msg for upgrading to a paid plan to access Github pages for private repositories. But for my fellow friends Github pages is working even in the case of private repository without any paid plan. How can this be resolved?
<img width="611" alt="image" src="https://github.com/community/community/assets/103261868/e7169cd7-3e8c-44e9-b050-a9fd14196a44">

**Comment by yoannchaudet** [comment]
On a free plan GitHub Pages is only available in public repositories.
